samples/scaleout/distribution-strategy/Core_6/Client/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.SenderSideScaleOut.Client";
+        var endpointConfiguration = new EndpointConfiguration("Samples.SenderSideScaleOut.Client");
+        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.SendFailedMessagesTo("error");
+
+        #region Logical-Routing
+
+        var routing = endpointConfiguration.UnicastRouting();
+        routing.RouteToEndpoint(typeof(DoSomething), "Samples.SenderSideScaleOut.Server");
+        //Distribute all messages using weighted algorithm
+        routing.Mapping.SetMessageDistributionStrategy(new WeightedDistributionStrategy(), t => true);
+
+        #endregion
+
+        #region File-Based-Routing
+
+        var transport = endpointConfiguration.UseTransport<MsmqTransport>();
+        transport.DistributeMessagesUsingFileBasedEndpointInstanceMapping("routes.xml");
+
+        #endregion
+
+        var endpointInstance = await Endpoint.Start(endpointConfiguration)
+            .ConfigureAwait(false);
```

Comment:
after the confusion from several people reading our doco we went with "best practices"